From d6f105694e6e19c7447d8ea6e562a7bbd88c0c88 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Vincent=20Stehl=C3=A9?= <vincent.stehle@arm.com>
Date: Mon, 30 Jan 2023 11:40:44 +0100
Subject: [PATCH] Add LDFLAGS for binaries
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Vincent Stehlé <vincent.stehle@arm.com>
---
 Makefile | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/Makefile b/Makefile
index 7d471da..3c187e1 100644
--- a/Makefile
+++ b/Makefile
@@ -88,31 +88,31 @@ HelloWorld.so: lib/lib-efi.a
 ShimReplace.so: lib/lib-efi.a
 
 cert-to-efi-sig-list: cert-to-efi-sig-list.o lib/lib.a
-	$(CC) $(ARCH3264) -o $@ $< lib/lib.a -lcrypto
+	$(CC) $(ARCH3264) -o $@ $< lib/lib.a $(LDFLAGS) -lcrypto
 
 sig-list-to-certs: sig-list-to-certs.o lib/lib.a
-	$(CC) $(ARCH3264) -o $@ $< lib/lib.a -lcrypto
+	$(CC) $(ARCH3264) -o $@ $< lib/lib.a $(LDFLAGS) -lcrypto
 
 sign-efi-sig-list: sign-efi-sig-list.o lib/lib.a
-	$(CC) $(ARCH3264) -o $@ $< lib/lib.a -lcrypto
+	$(CC) $(ARCH3264) -o $@ $< lib/lib.a $(LDFLAGS) -lcrypto
 
 hash-to-efi-sig-list: hash-to-efi-sig-list.o lib/lib.a
-	$(CC) $(ARCH3264) -o $@ $< lib/lib.a
+	$(CC) $(ARCH3264) -o $@ $< lib/lib.a $(LDFLAGS)
 
 cert-to-efi-hash-list: cert-to-efi-hash-list.o lib/lib.a
-	$(CC) $(ARCH3264) -o $@ $< lib/lib.a -lcrypto
+	$(CC) $(ARCH3264) -o $@ $< lib/lib.a $(LDFLAGS) -lcrypto
 
 efi-keytool: efi-keytool.o lib/lib.a
-	$(CC) $(ARCH3264) -o $@ $< lib/lib.a
+	$(CC) $(ARCH3264) -o $@ $< lib/lib.a $(LDFLAGS)
 
 efi-readvar: efi-readvar.o lib/lib.a
-	$(CC) $(ARCH3264) -o $@ $< lib/lib.a -lcrypto
+	$(CC) $(ARCH3264) -o $@ $< lib/lib.a $(LDFLAGS) -lcrypto
 
 efi-updatevar: efi-updatevar.o lib/lib.a
-	$(CC) $(ARCH3264) -o $@ $< lib/lib.a -lcrypto
+	$(CC) $(ARCH3264) -o $@ $< lib/lib.a $(LDFLAGS) -lcrypto
 
 flash-var: flash-var.o lib/lib.a
-	$(CC) $(ARCH3264) -o $@ $< lib/lib.a
+	$(CC) $(ARCH3264) -o $@ $< lib/lib.a $(LDFLAGS)
 
 clean:
 	rm -f PK.* KEK.* DB.* $(EFIFILES) $(EFISIGNED) $(BINARIES) *.o *.so
-- 
2.35.1

